_translator-files/po/ru/Running-a-Server.po

```diff
@@ -88,10 +89,9 @@ msgstr "Нужно ли мне запускать сервер?"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:23
-#, fuzzy, no-wrap
-#| msgid "**No**. You don't need to run a Server. You can use the Servers listed by the built-in Directories and use Jamulus without running a Server yourself. You can also use an unlisted Server if you know its internet address. Or you can use a third party hosting service such as [melomax](https://melomax.live/jamulus-hosting/) or [KOORD](https://koord.live/). There's probably a Server nearby that you and your friends can use at low enough latency for most needs.\n"
+#, no-wrap
 msgid "**No**. You don't need to run a Server. You can use the Servers listed by the built-in Directories and use Jamulus without running a Server yourself. You can also use an unlisted Server if you know its internet address. Or you can use a third party hosting service such as [melomax](https://melomax.live/jamulus-hosting/). There's probably a Server nearby that you and your friends can use at low enough latency for most needs.\n"
-msgstr "**Нет**. Вам не нужно запускать сервер. Вы можете использовать серверы, перечисленные во встроенных каталогах, и пользоваться Jamulus без запуска сервера. Вы также можете использовать сервер, не внесенный в список, если знаете его интернет-адрес. Или вы можете воспользоваться услугами стороннего хостинга, например [melomax](https://melomax.live/jamulus-hosting/) или [KOORD](https://koord.live/). Вероятно, поблизости есть сервер, который вы и ваши друзья сможете использовать с достаточно низкой задержкой для большинства нужд.\n"
+msgstr "**Нет**. Вам не нужно запускать сервер. Вы можете использовать серверы, перечисленные во встроенных каталогах, и пользоваться Jamulus без запуска сервера. Вы также можете использовать сервер, не внесенный в список, если знаете его интернет-адрес. Или вы можете воспользоваться услугами стороннего хостинга, например [melomax](https://melomax.live/jamulus-hosting/) . Вероятно, поблизости есть сервер, который вы и ваши друзья сможете использовать с достаточно низкой задержкой для большинства нужд.\n"
```

Comment:
Extra space after `https://melomax.live/jamulus-hosting/)` and the following full stop.